src/aura/CRLP_FilterGroup/CRLP_FilterGroupHelper.js

```diff
@@ -231,7 +344,7 @@
      * @description: opens a modal popup so user can add or edit a filter rule
      */
     resetActiveFilterRule: function(cmp) {
-        var defaultFilterRule = {objectName: '', fieldName: '', operatorName: '', constant: ''};
+        var defaultFilterRule = {objectName: '', fieldName: '', operation: '', constant: ''};
```

Comment:
We should rename constant to value or valueName here.

Reply:
Yes. Good catch!